tests/message_ut/message_ut.c

```diff
@@ -165,49 +169,46 @@ TEST_FUNCTION(when_allocating_memory_for_the_message_fails_then_message_create_f
 TEST_FUNCTION(message_clone_with_a_valid_argument_succeeds)
```

Comment:
Agree, I will add all the tests as a separate PR, I am simply converting now to umock_c.

---
In reply to: [104191421](https://github.com/Azure/azure-uamqp-c/pull/28#discussion_r104191421) [](ancestors = 104191421)